electron/Transport.js

```diff
@@ -0,0 +1,16 @@
+const Transport = require('winston-transport');
+const stripAnsi = require('strip-ansi');
+
+module.exports = class CustomTransport extends Transport {
+  constructor(opts) {
+    super(opts);
+    this.loggedMessages = opts.loggedMessages;
+  }
+
+  log(info, callback) {
+    const level = stripAnsi(info.level);
+    this.loggedMessages.push(`${info.timestamp} [${level}]: ${info.message}`);
+    // Perform the writing to the remote service
+    callback();
+  }
+};
```

Comment:
DP1 - Could we rename the CustomTransport class to be a little more informative? Something like InMemoryTransport – whatever you think gets across the spirit of what this transport is accomplishing. 

DP2 - Could you also rename the file to be the name of the class we define, whatever that ends up being in resolving DP1. This will require updating imports as well.